npm test does not work on Windows 7

Output is as follows:

```
C:\git\gulp-merge-json\node_modules\.bin\_mocha.CMD:1
(function (exports, require, module, __filename, __dirname) { @IF EXIST "%~dp0
                                                              ^
SyntaxError: Unexpected token ILLEGAL
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at runFn (C:\git\gulp-merge-json\node_modules\istanbul\lib\command\common\ru
n-with-cover.js:122:16)
    at Object.run (C:\git\gulp-merge-json\node_modules\istanbul\lib\command\comm
on\run-with-cover.js:254:9)
    at TestCommand.Command.mix.run (C:\git\gulp-merge-json\node_modules\istanbul
\lib\command\test.js:27:22)
    at runCommand (C:\git\gulp-merge-json\node_modules\istanbul\lib\cli.js:78:19
)
npm ERR! Test failed.  See above for more details.
```

This is caused by __mocha_ not translating into a javascript files in Windows

See istanbul/issues/90

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "merge"
   ],
   "scripts": {
-    "test": "istanbul test _mocha --report html -- test/*.js --reporter spec",
+    "test": "istanbul test ./node_modules/mocha/bin/_mocha --report html -- test/*.js --reporter spec",
     "coveralls": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage"
   },
   "dependencies": {

